backend/src/vscode-output.ts

```diff
@@ -0,0 +1,36 @@
+import * as vscode from "vscode";
+import * as _ from "lodash";
+import stripAnsi = require("strip-ansi");
+import { Output } from "./output";
+
+
+export class OutputChannel implements Output {
```

Comment:
I would change name. 
1. vscode has the same class name.
2. I see it has a list of channels.

Reply:
GeneratorOutput